Makefile.osx

```diff
@@ -37,30 +37,28 @@ ifneq ($(strip $(shell which xcpretty)),)
 endif
 
 release:
-	mkdir build.release;                               \
-    cd build.release;                                  \
-    cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
-          -DCMAKE_BUILD_TYPE=RELEASE \
-          -DCMAKE_BUILD_NUMBER="${BUILD_NUMBER}" \
-          -DMUSESCORE_BUILD_CONFIG="${MUSESCORE_BUILD_CONFIG}" \
-          -DMUSESCORE_REVISION="${MUSESCORE_REVISION}" \          
-          -DTELEMETRY_TRACK_ID="${TELEMETRY_TRACK_ID}" \
-          -DBUILD_UI_MU4="${BUILD_UI_MU4}" \
-	    -DBUILD_VST="${BUILD_VST}" \
-	    -DVST3_SDK_PATH="${VST3_SDK_PATH}" \
-          -DMODULE_BUILD_PCH="${MODULE_BUILD_PCH}" \
-          .. -G Xcode; 			       \
-    xcodebuild -project ${XCODEPROJ} -target lrelease;                            \
-		$(FORMATTER_START) xcodebuild -project ${XCODEPROJ} -configuration Release -target ALL_BUILD $(FORMATTER_END);    \
+	@echo "Make release"; \
+	mkdir build.release; \
+	cd build.release; \
+	cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+	-DCMAKE_BUILD_TYPE=RELEASE \
+	-DCMAKE_BUILD_NUMBER="${BUILD_NUMBER}" \
+	-DMUSESCORE_BUILD_CONFIG="${MUSESCORE_BUILD_CONFIG}" \
+	-DMUSESCORE_REVISION="${MUSESCORE_REVISION}" \
+	-DTELEMETRY_TRACK_ID="${TELEMETRY_TRACK_ID}" \
+	.. -G Xcode; \
```

Comment:
I'd rather indent all those options to `cmake`

Reply:
fixed